sdc/datatypes/common_functions.py

```diff
@@ -131,3 +132,19 @@ def _append_list_string_array_impl(A, B):
                 return new_data
 
             return _append_list_string_array_impl
+
+
+@numba.njit
+def _hpat_ensure_array_capacity(new_size, arr):
+    '''Function creating a copy of numpy array with a size more than specified'''
+    # TODO: replace this function with np.resize when supported by Numba
```

Comment:
`np.resize` resizes array in other way than this algorithm. The algorithm can generates array of size more than `new_size` and doesn't repeat values of `arr` in the resulting array. Looks like there is mismatch somewhere :)

Reply:
Removed the comment at all)